server/apiCalls/transformers/memos.test.js

```diff
@@ -13,10 +13,16 @@ const offering_SF1625_202121 = {
   courseCode: 'SF1625',
   offeringId: '1',
```

Comment:
`offeringId` has to be removed to be sure tests working properly

Reply:
Removed